napari/layers/image/tests/test_image.py

```diff
@@ -206,7 +206,7 @@ def test_error_non_rgb_image():
     np.random.seed(0)
     data = np.random.random(shape)
     with pytest.raises(ValueError):
-        layer = Image(data, rgb=True)
+        Image(data, rgb=True)
```

Comment:
_ = 
```suggestion
        _ = Image(data, rgb=True)
```

Reply:
Eh, ignore me, just saw that it's not supposed to even get to the assignment step! :joy: